.circleci/config.yml

```diff
@@ -2,6 +2,7 @@ version: 2.1
 orbs:
     node: circleci/node@5.0.2
     browser-tools: circleci/browser-tools@1.4.0
+    codecov: codecov/codecov@3.2.3
```

Comment:
How did you verify this orb version is using the non-deprecated codecov uploader? I'm having a little trouble connecting all the dots here.

Reply:
I removed the deprecated npm package (and npm task `codecov`), replaced it with the `codecov/upload` step. Have a look at the CircleCI report for this PR, too.

Reply:
Is the "codecov" npm dependency still needed in pkg json with this change?

Reply:
No, it's not needed anymore.